docs/swarm/join-nodes.md

```diff
@@ -100,29 +105,29 @@ $ docker swarm join \
 This node joined a swarm as a manager.
 ```
 
-<!--TODO WIP
-Manager nodes use the listen address for cluster management communications. The
-other nodes on the swarm must be able to access the manager node on the
-IP address and port you specify for the listen address.
-
-Especially when there are multiple active network interfaces, you should
-you explicitly define the listen address when you add a manager node to the a
-swarm:
-
-  ```bash
-  docker swarm join \
-  --token <MANAGER-TOKEN> \
-  --listen-addr <NODE-IP>:<PORT> \
-  <MANAGER-IP>:<PORT>
-  ```
-this will change for https://github.com/docker/docker/pull/24237 ->>
-Replace <NODE-IP> with the IP address of the node that is joining the swarm.
-Replace <MANAGER-IP> with the address of the swarm manager.
-
-Only manager nodes use the listen address. If you specify `--listen-addr` for a
-worker node, the node only uses the listen address if it is promoted to a
-manager.
--->
+### Advertise address
+
+Manager nodes use an **advertise address** to allow other nodes in the swarm
+access to the API and overlay networking. If the system has multiple IP
+addresses, you must specify the correct  `--advertise-addr` to enable
+inter-manager communication and overlay networking:
```

Comment:
It should work fine without `--advertise-addr`. The address is automatically detected by the manager it connects to. We provide that option for join but I actually can't think of a use case where it would make sense to use it.